File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int,int>mpp;
        vector<int>ans;
        for(int i=0;i<nums.size();i++)
        {
            mpp[nums[i]]++;
            if(mpp[nums[i]]!=1)
            {
                ans.push_back(nums[i]);
            }
        }
        return ans;
    }
};

File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let freq={};
    let ans=[]
    for(let i=0;i<nums.length;i++)
    {
        if(!freq[nums[i]])
            {
                freq[nums[i]]=0;
            }
            freq[nums[i]]++;

        if(freq[nums[i]]!=1)
        {
            ans.push(nums[i]);
        }
    }
    return ans;
};